data-prepper-core/src/test/java/org/opensearch/dataprepper/acknowledgements/DefaultAcknowledgementSetTests.java

```diff
@@ -136,13 +136,11 @@ void testDefaultAcknowledgementInvalidAcquire() {
         assertThat(invalidAcquiresCounter, equalTo(1));
     }
 
-    @Test
     void testDefaultAcknowledgementInvalidRelease() {
         defaultAcknowledgementSet.add(event);
         defaultAcknowledgementSet.complete();
         DefaultAcknowledgementSet secondAcknowledgementSet = createObjectUnderTest();
         assertThat(defaultAcknowledgementSet.release(handle2, true), equalTo(false));
-        assertThat(invalidReleasesCounter, equalTo(1));
```

Comment:
Should we assert this is equal to 0?

Reply:
Actually, the counter should be removed.